keycodes.go

```diff
@@ -4,22 +4,24 @@ package promptui
 
 import "github.com/chzyer/readline"
 
+// These runes are used to identity the commands entered by the user in the command prompt. They map
+// to specific actions of prompt-ui and can be remapped if necessary.
 var (
-	// KeyEnter is the default key for submission/selection
+	// KeyEnter is the default key for submission/selection inside a command line prompt.
```

Comment:
This seems unnecessary. Everything here is inside a command line prompt :)

Reply:
Good catch